release-notes.txt

```diff
@@ -1,3 +1,772 @@
+RIOT-2020.04 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2020.04 release includes:
+
+- Add support for 6LoWPAN IPv6 extension header next header compression
+- Add support for DHCPv6 prefix delegation client
+- Add support for kw41zrf and at86rf215 IEEE 802.15.4 radios
+- Improvements on power management for ESP32
+- Improvements and extensions in support of fe310 and lpc2387
+- lwIP: provide support for asynchronous sock
+- Improvements on Xtimer module and testing
+- Initial addition of the new timer subsystem: Ztimer
+- Migration to Kconfig - phase I: add multiple GNRC and system modules
+- +10 new boards, +11 new drivers, +7 new packages
+
+
+About 555 pull requests, composed of 1514 commits, have been merged since the
+last release, and about 22 issues have been solved. 60 people contributed with
+code in 102 days. Approximately 3346 files have been touched with 358398 (+)
+insertions and 184720 deletions (-).
```

Comment:
Very precise numbers while using words like "about" and "approximately" ;-)

Reply:
Done